src/haddock/modules/refinement/openmm/openmm.py

```diff
@@ -0,0 +1,308 @@
+"""OpenMM module."""
```

Comment:
```suggestion
"""
Refinement module using OpenMM.

This module uses OpenMM to refine the models before exploring docking in-depth.
We have created a generalized refinement protocol using OpenMM for which
you can modify a minimal set of parameters.

To get a list of all possible parameters, run::

     haddock3-cfg -m openmm
     
This module will refine all models coming from the previous workflow step and send them
to the next step in the workflow. If you want to use other modules such as `flexref` or `emref` after
the OpenMM module, you need to recreate the topologies by simply adding a `[topoaa]` step in the workflow.
See examples in `examples/thirdparty/openmm` folder.
"""
```

Please help me further here to add a short paragraph explaining the major concepts of the implementation.